### ISSUE TITLE
Episode Error

### ISSUE THREAD

**Issue by SmartOinker** [Jayapraveen/ITPro.Tv-Courses-downloader#15]
```
(VirtualEnv) D:\Documents\Others\Tools and Resources\ITPro.Tv-Courses-downloader-master\ITPro.Tv-Courses-downloader-master>python itprotv.py
Token is Valid!

0 Get Started
1 Most Popular
2 CompTIA
3 Microsoft
4 Cisco
5 Security Skills
6 Linux
7 (ISC)²
8 EC-Council
9 Python
10 Apple
11 AWS
12 VMware
13 ITIL®
14 Google
15 Agile
16 CertNexus
17 Service Management
18 Project Management
19 PMI®
20 PRINCE2®
21 Big Data
22 GIAC
23 ISACA
24 Governance
25 Six Sigma
26 Automation
27 Networking Skills
28 Foundation Skills
29 Cloud Technology Skills
30 Virtualization Skills
31 OfficeProTV
32 DevProTV
33 Podcasts
34 Webinars
35 Business Skills
36 Legacy

Enter your choice: 8
0 CEH v10
1 CHFI v9
2 ECES
3 EC-Council Certified Incident Handler v2
4 Certified Ethical Hacker v11 (In Production)

Enter your choice: 0
0 Overview
1 Intro to Ethical Hacking: Threats
2 Intro to Ethical Hacking: Threats Part 2
3 Intro to Ethical Hacking: Hacking Concepts
4 Intro to Ethical Hacking: Hacking Concepts Part 2
5 Intro to Ethical Hacking: Security Controls
6 Intro to Ethical Hacking: Security Controls Part 2
7 Intro to Ethical Hacking: Security Controls Part 3
8 Intro to Ethical Hacking: Security Controls Part 4
9 Intro to Ethical Hacking: Security Controls Part 5
10 Intro to Ethical Hacking: Pentesting
11 Intro to Ethical Hacking: Laws and Standards
12 Footprinting
13 Footprinting Part 2
14 Footprinting Part 3
15 Host Discovery
16 Scanning with Nmap
17 Scanning with Nmap Part 2
18 ProxyChains
19 Enumeration
20 Vulnerability Analysis Concepts
21 Vulnerability Analysis Tools
22 Password Attacks
23 Password Attacks Part 2
24 Password Attacks Part 3
25 Password Attacks Part 4
26 Password Attacks Part 5
27 Privilege Escalation
28 Privilege Escalation Part 2
29 Privilege Escalation Part 3
30 Covert Data Gathering
31 Hidden Files
32 Covering Tracks
33 Malware Threats cehv10
34 Malware Threats Part 2
35 Malware Threats Part 3
36 Network Sniffing
37 Network Sniffing Part 2
38 Social Engineering
39 Social Engineering Part 2
40 Social Engineering Part 3
41 Denial of Service
42 Denial of Service Part 2
43 Denial of Service Part 3
44 Session Hijacking
45 Session Hijacking Part 2
46 Session Hijacking Part 3
47 Evading IDS, Firewalls and Honeypots
48 Evading IDS, Firewall and Honeypots Part 2
49 Evading IDS, Firewall and Honeypots Part 3
50 Evading IDS, Firewall and Honeypots Part 4
51 Evading IDS, Firewall and Honeypots Part 5
52 Hacking Web Servers
53 Common Web App Threats
54 Common Web App Threats Part 2
55 Common Web Apps Threats Part 3
56 Common Web App Threats Part 4
57 Practical Web App Hacking
58 Practical Web App Hacking Part 2
59 Practical Web App Hacking Part 3
60 SQL Concepts
61 SQL Injection Types
62 Other SQLi and Tools
63 Wireless Hacking Concepts
64 Wireless Hacking: Tools
65 Wireless Hacking: Tools Part 2
66 Wireless Hacking: Common Threats
67 Wireless Hacking: Common Threats Part 2
68 Wireless Hacking: Cracking WEP
69 Wireless Hacking: Cracking WPA/WPA2
70 Mobile Hacking
71 IoT Concepts
72 IoT Concepts Part 2
73 IoT Attacks
74 IoT Attacks Part 2
75 IoT Attacks Part 3
76 Cloud Computing Concepts
77 Cloud Computing Concepts Part 2
78 Cloud Computing Attacks
79 Cloud Computing Attacks Part 2
80 Cryptography Concepts
81 Cryptography Concepts Part 2
82 Cryptography Concepts Part 3
83 Cryptography Concepts Part 4
84 Cryptography Concepts Part 5
85 Cryptography Concepts Part 6
86 Cryptography Attacks
Total= 87
Traceback (most recent call last):
  File "D:\Documents\Others\Tools and Resources\ITPro.Tv-Courses-downloader-master\ITPro.Tv-Courses-downloader-master\itprotv.py", line 148, in <module>
    episodes_download_links = get_download_links(episode_links,data_auth,resolution)
  File "D:\Documents\Others\Tools and Resources\ITPro.Tv-Courses-downloader-master\ITPro.Tv-Courses-downloader-master\itprotv.py", line 109, in get_download_links
    if (req_data["episode"][resolution] == None):
KeyError: 'episode'
```
Hi I got the above error. Would it be possible to give some solutions? Thank you.

**Comment by sagary2j** [comment]
I am also facing same error. Any help!!

**Comment by flyingdrnick** [comment]
Req_data seems to be having issue from the requests.get

**Comment by Jayapraveen** [comment]
Fixed in commit add8838.

**Comment by Ani07-05** [comment]
I am still getting the same error
Pls do the needful
